content/en/docs/release_notes/austen.md

```diff
@@ -6,11 +6,14 @@ date: 2022-09-09
 Hide_readingtime: true
 ---
 ## Summary
-Brief summary of the notable release contents.
+
+{{% variables/apibuilder_prod_name %}} v5 is released. Projects created on v4 are compatible with v5, but require installing upgrades for components (see [Upgrades](/docs/developer_guide/console#updates-tab)).
```

Comment:
Link out to the new set of docs you are righting now?

Reply:
You'll want to add the comment on staging

Reply:
left issue to resolve in staging